client/src/hooks/datafiles/mutations/useCompress.ts

```diff
@@ -73,18 +69,15 @@ function useCompress() {
     compressionType: string;
     fromDownload?: boolean;
   }) => {
-    dispatch({
-      type: 'DATA_FILES_SET_OPERATION_STATUS',
-      payload: { status: 'RUNNING', operation: 'compress' },
-    });
+    setStatus({ type: 'RUNNING' });
 
     let defaultPrivateSystem: TPortalSystem | undefined;
 
     if (files[0].scheme === 'private' && files[0].api === 'tapis') {
       defaultPrivateSystem = undefined;
     }
 
-    if (!defaultAllocation) {
+    if (!allocationForCompress) {
       throw new Error('You need an allocation to compress.', {
```

Comment:
Not for this PR, but we need a follow on ticket. As we need the UI to be updated if we don't have an allocation here and in useExtract.

Reply:
https://tacc-main.atlassian.net/browse/WG-504